cartographer/mapping/trajectory_node.h

```diff
@@ -67,8 +66,6 @@ struct TrajectoryNodes {
   std::vector<mapping::TrajectoryNode> trajectory_nodes;
 };
 
-proto::Trajectory ToProto(const std::vector<TrajectoryNode>& nodes);
```

Comment:
As @SirVer said, once this PR is LGTM, I'll follow up with one for `cartographer_ros`, and then somebody will hit the merge buttons very quickly.